src/js/components/common/left-nav.js

```diff
@@ -35,17 +36,25 @@ export const LeftNav = ({ sections }) => {
       {sections.map(({ itemClass = '', items = [], title = '' }, index) => (
         <React.Fragment key={`${itemClass}-${index}`}>
           <ListSubheader disableSticky={true}>{title}</ListSubheader>
-          {items.map(({ exact, path, secondaryAction = null, style = {}, title = '' }) => (
+          {items.map(({ exact, path, secondaryAction = null, style = {}, title = '', url }) => (
```

Comment:
with the added differentiation between regular `a` and `NavLink` there is an additional error in the browser console, since `a` doesn't understand the boolean `exact`, but `NavLink` also complains about this being just a string - so you could either have `exact={url ? `${exact}` : exact} or:
```suggestion
          {items.map(({ exact, path, secondaryAction = null, style = {}, title = '', url }) => {
            const props = url
              ? { component: 'a', exact: `${exact}`, href: url, rel: 'noopener noreferrer', target: '_blank', to: url }
              : { component: NavLink, exact, to: path };
            return (
              <ListItem className={`navLink ${itemClass} ${classes.list}`} key={path} secondaryAction={secondaryAction} style={style} {...props}>
                <ListItemText className={classes.listItem} primary={title} />
              </ListItem>
            );
          })}
```
- this would keep the `url`-based decisions all in one place...

Reply:
Thanks, have updated with these adjustments!